elife-1234567890-v2/elife-1234567890-v2.xml

```diff
@@ -1973,41 +1991,30 @@ module load anaconda</code>
                     </aff>
                 </contrib>
             </contrib-group>
-            <related-object id="sa0ro1" object-id="10.1101/2021.11.09.467796" object-id-type="id" link-type="continued-by" xlink:href="https://sciety.org/articles/activity/10.1101/2021.11.09.467796"/>
+            <kwd-group kwd-group-type="Importance-of-claim">
+                <!-- Controlled vocabulary of importance values: Landmark; Fundamental; Important; Noteworthy; Useful; Flawed -->
+                <kwd><ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0/">Landmark</ext-link></kwd>
+            </kwd-group>
+            <kwd-group kwd-group-type="Strength-of-evidence">
+                <!-- Controlled vocabulary of strength values: Tour-de-force; Compelling; Convincing; Solid; Incomplete; Inadequate -->
+                <kwd>Compelling</kwd>
+            </kwd-group>
+            <!-- remove sciety related object? Update? -->
+            <!-- <related-object id="sa0ro1" object-id="10.1101/2021.11.09.467796" object-id-type="id" link-type="continued-by" xlink:href="https://sciety.org/articles/activity/10.1101/2021.11.09.467796"/> -->
         </front-stub>
         <body>
-            <p>This is an Editor's evaluation, which is a summary of the article provided by the BRE. It will only contain one or more paragraphs, no figures, tables or videos.</p>
+            <p>This is an Editor's summary, which is a summary of the article provided by the BRE. It will only contain one or more paragraphs, no figures, tables or videos.</p>
```

Comment:
Can we add content that has the controlled vocabulary in it, like in [this example](https://epp-node-prototype.elifesciences.org/article/10.1101/2022.04.13.488149/reviews)?

I think we also need to consider what tagging is required on the vocab words in the paragraph, maybe `named-content`.